### ISSUE TITLE
.mul data doesn't transfer properly after finishing a mission

### ISSUE THREAD

**Issue by RyuWanderfalke** [MegaMek/mekhq#1709]
### Environment
What version of MekHQ does your issue apply to? 0.47.5
What operating system are you using?  Windows 10
What java version are you using?  Java version 1.8.0_251

### Description
I changed the parameters of the bot setup after starting a mission from the Against the Bot campaign for the first time, when being in the MegaMek setup. After finishing the mission, I saved the .mul file as usual (which I have never needed up to this point) and when it went to the usual "Did your side control the battlefield?" message box, nothing happened after klicking "Yes". I then clicked "Resolve manually" and when it asked me about the mul-file, nothing happened after selecting the file and clicking "ok". I thought that it might have been the result of me tempering with the bot and enemy unit setup (the idea was to speed things up since the bot took forever making its turns with 15 fast moving mechs in the previous mission), but it turns out, after finishing the next mission WITHOUT tempering with the enemy force and bot setup, the same thing as described before happened again! Any idea how to fix this? Proceeding without the consequences of each battle is rather dull and pointless...
![bug](https://user-images.githubusercontent.com/64443549/80802155-d4dc5000-8bae-11ea-88b7-9e38498a3448.jpg)

[CampaignAndCustoms2.zip](https://github.com/MegaMek/mekhq/files/4563968/CampaignAndCustoms2.zip)

[relevant mul file.zip](https://github.com/MegaMek/mekhq/files/4564210/relevant.mul.file.zip)

[megameklog.txt](https://github.com/MegaMek/mekhq/files/4563971/megameklog.txt)
[mekhqlog.txt](https://github.com/MegaMek/mekhq/files/4563973/mekhqlog.txt)

**Comment by Windchild292** [comment]
My apologies on not getting back to you on this. I must have forgotten to attach the workaround mul, and I further apologize for that.

The underlying cause has been fixed for 0.47.6, and running this campaign in 0.47.6 will resolve the issue in question. I actually used this campaign to perform my release verification for the fix.

My apologies again for the delay in my response, and closing this as resolved in 0.47.6
